sources/api/datastore/src/serialization/mod.rs

```diff
@@ -228,6 +228,14 @@ mod test {
 
     #[test]
     fn ok_key() {
+        let serializer = MapKeySerializer::new();
+        let m = TestEnum::FooBar;
+        let res = m.serialize(&serializer).unwrap();
+        assert_eq!(res, "FooBar");
+    }
+
+    #[test]
+    fn ok_enum_key() {
```

Comment:
After rebasing, you shouldn't have any changes in the datastore. Probably it is these lines that are breaking the build.

Reply:
Removed these lines - thanks! :)